instrumentation/kotlinx-coroutines/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/kotlinxcoroutines/KotlinCoroutinesInstrumentationHelper.java

```diff
@@ -13,11 +13,16 @@
 
   public static CoroutineContext addOpenTelemetryContext(CoroutineContext coroutineContext) {
     Context current = Context.current();
+    return addOpenTelemetryContext(coroutineContext, current);
+  }
+
+  public static CoroutineContext addOpenTelemetryContext(
```

Comment:
Might be missing it but looks like we can revert this file? Don't think I see this used

Reply:
Thanks, reverted